contracts/ButtonToken.sol

```diff
@@ -544,7 +544,7 @@ contract ButtonToken is IButtonToken, Initializable, OwnableUpgradeable {
         bool valid;
         (newPrice, valid) = IOracle(oracle).getData();
 
-        return (valid ? newPrice : lastPrice, valid);
```

Comment:
Maybe add a comment here on why this check exists?

Reply:
good idea